protocol/block_test.go

```diff
@@ -223,7 +224,7 @@ func newTestChain(tb testing.TB, ts time.Time) (c *Chain, b1 *bc.Block) {
 	if err != nil {
 		testutil.FatalErr(tb, err)
 	}
-	c, err = NewChain(ctx, b1.Hash(), memstore.New(), nil)
+	c, err = NewChain(ctx, b1.Hash(), prottest.NewMemStore(), nil)
```

Comment:
this newTestChain func is moot if we're moving it into protocol_test. We can use `prottest.NewChain` and friends.